File: src/main/java/com/team573/gongguri/domain/groupPurchase/dto/GroupPurchaseWithChatResponseDto.java
package com.team573.gongguri.domain.groupPurchase.dto;

import java.time.LocalDateTime;
import lombok.Builder;

@Builder
public record GroupPurchaseWithChatResponseDto(
    Long id,
    String title,
    Integer maxParticipants,
    Long participantCount,
    String progressStatus,
    String imageUrl,
    String chatMessage,
    LocalDateTime createAt
) {

}

File: src/main/java/com/team573/gongguri/domain/grouppurchase/dto/GroupPurchaseWithChatResponseDto.kt
package com.team573.gongguri.domain.grouppurchase.dto

import java.time.LocalDateTime

data class GroupPurchaseWithChatResponseDto(
    val id: Long,
    val participantId: Long,
    val title: String,
    val maxParticipants: Int,
    val participantCount: Long,
    val progressStatus: String,
    val imageUrl: String,
    val chatMessage: String?,
    val createAt: LocalDateTime
)
